package.json

```diff
@@ -43,11 +44,16 @@
     "ws": "^6.1.2"
   },
   "devDependencies": {
+    "eslint-plugin-jest": "^22.2.2",
+    "jest": "^24.0.0",
     "serverless-content-encoding": "^1.1.0",
     "serverless-domain-manager": "^2.6.10",
     "serverless-log-forwarding": "^1.3.0",
     "serverless-offline": "^3.31.3",
     "serverless-plugin-tracing": "^2.0.0",
     "serverless-ssm-fetch": "^1.0.1"
+  },
+  "jest": {
```

Comment:
You might want to put this in the jest.config instead of putting it here in the package.json 🌮

Reply:
I was going off of this:
https://github.com/Microsoft/vscode-recipes/tree/master/debugging-jest-tests

But I'll look into getting it added to the `jest.config`. Thanks @corymcdonald! 👍

Reply:
@corymcdonald 
> Jest's configuration can be defined in the package.json file of your project or through the --config <path/to/json> option. If you'd like to use your package.json to store Jest's config, the "jest" key should be used on the top level so Jest will know how to find your settings:

Does this mean we can put `testEnvironment` in the `jest.config` then?

Reply:
Nevermind. Appears so :)
https://github.com/facebook/jest/blob/master/jest.config.js#L31

Reply:
Fixed: e44e382

I actually already had it in the `jest.config`: https://github.com/grahamperich/ethereum-dex-prices-service/pull/15/files#diff-2d0cd5d10b9604941c38c6aac608178aR136